src/main/java/org/opensearch/neuralsearch/query/SparseEncodingQueryBuilder.java

```diff
@@ -99,6 +104,7 @@ protected void doXContent(XContentBuilder xContentBuilder, Params params) throws
         xContentBuilder.startObject(fieldName);
         xContentBuilder.field(QUERY_TEXT_FIELD.getPreferredName(), queryText);
         xContentBuilder.field(MODEL_ID_FIELD.getPreferredName(), modelId);
+        if (null != maxTokenScore) xContentBuilder.field(MAX_TOKEN_SCORE_FIELD.getPreferredName(), maxTokenScore);
         printBoostAndQueryName(xContentBuilder);
```

Comment:
why do we need to print ?

Reply:
max_token_score is a parameter for this query clause, and it should be treated same as other parameters

Reply:
This is similar to exist code: https://github.com/opensearch-project/neural-search/blob/main/src/main/java/org/opensearch/neuralsearch/query/NeuralQueryBuilder.java#L120, I think this is for debugging purpose.

Reply:
The print is not logging, but output to XContent. The boost and query name are also parameters and should be included in the XContent